firestore.rules

```diff
@@ -2,35 +2,43 @@ rules_version = '2';
 
 service cloud.firestore {
   match /databases/{database}/documents {
-    // TODO: Add function for auth check.
+    // Returns true if the user is logged in.
+    function loggedIn() {
```

Comment:
can this be called isLoggedIn()?

Reply:
leaving as loggedIn as discussed in-person